README.md

```diff
@@ -335,6 +335,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [MicroBadger][microbadger] - Analyze the contents of images and add metadata labels
 * [packer](https://www.packer.io/docs/builders/docker.html) - Hashicorp tool to build machine images including docker image integrated with configuration management tools like chef, puppet, ansible
 * [portainer](https://github.com/duedil-ltd/portainer) - Apache Mesos framework for building Docker images by [@duedil-ltd](https://github.com/duedil-ltd)
+* [runlike](https://github.com/lavie/runlike) - Generate `docker run`command and options from running containers by [@lavie](https://github.com/lavie)
```

Comment:
According to the project description:

>This is very much a work in progress. Many docker run options aren't yet supported, but the most commonly used ones are. Feel free to send pull requests if you add any or if you happen to fix any (of the many) bugs this package undoubtedly has.
>
>Probably shouldn't use this in production yet. If you do, double check that it's actually running what you want it to run.

Please add in the description that it's a WIP, or putting a symbol after the link is better like

```
[runlike](https://github.com/lavie/runlike) :face_with_head_bandage: - Generate
```

Reply:
:nerd_face: symbol for CLI tools is missing, too.

```
[runlike](https://github.com/lavie/runlike) :face_with_head_bandage: :nerd_face: - Generate
```